tools/test.py

```diff
@@ -176,8 +176,7 @@ def process_product(i, product):
 
 
 ontobee = 'http://www.ontobee.org/browser/rdf.php?o=%s&iri=http://purl.obolibrary.org/obo/'
-ols = 'https://www.ebi.ac.uk/ols/ontologies/%s/terms?iri=http%%3A%%2F%%2Fpurl.obolibrary.org%%2Fobo%%2F'
-
+ols = 'https://www.ebi.ac.uk/ols4/ontologies/%s/entities/http%%253A%%252F%%252Fpurl.obolibrary.org%%252Fobo%%252F'
```

Comment:
@jamesaoverton I got the tests to pass. Note that I am a bit confused why I needed to _double encode_ the entity IRI here to get it to work, but I think its because they moved from "URL parameter" syntax (?iri=) to "URL path" syntax.

Reply:
Yes, moving from the query string into the path seems to require the extra escape. It's very ugly but it seems to work fine.

Reply:
I was just reviewing some OLS4 tickets and stumbled upon this again. We do support `?iri=`, `?shortForm=` and `?oboId=` (`?curie=`) for /terms and /entities. What we do not support on the frontend is `?id=`. If you encounter any discrepancies, let us know. Thanks.